src/resources/js/views/events-bar-inputs.js

```diff
@@ -97,7 +97,7 @@ tribe.events.views.eventsBarInputs = {};
 		$container
 			.find( obj.selectors.inputWrapper )
 			.each( function( index, wrapper ) {
-				var inputWrapperClass = wrapper.className.match( /tribe-common-c-search__input-control--[a-z]+/ );
+				var inputWrapperClass = wrapper.className.match( /tribe-events-c-search__input-control--[a-z]+/ );
```

Comment:
I really think we should completely remove the requirement for this particular wrapper class. Possibly only depend on a particular "simpler" data attribute.

Reply:
I agree, but I think we should tackle this in a separate ticket. In this, I'm just fixing something that should have been working but isn't.

Reply:
Sounds good to me. Would you mind logging that for the Backlog? and get it merged.